framer/scripts/templates/slider.txt

```diff
@@ -1,21 +1,21 @@
-{{=« »=}}
-import * as React from 'react';
-import { addPropertyControls, ControlType } from 'framer';
-import Mui«componentName» from '@material-ui/core/«componentName»';
+{{=« »=}}	
```

Comment:
Is this expected?

Reply:
This file should’t have changes will check it again